### PR TITLE
FWK-588 jake script runner

### PR DESCRIPTION
**Pull request by aprakash-gc**

### CODE REVIEW

File: jakelib/scripts.jake [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const { cyan, green, magenta, red } = require('chalk');
+
+const { question } = require('readline-sync');
+
+const CLUSTERS = {
```
Comment (bearfact) [inline jakelib/scripts.jake:5]: We should import this from a shared place, so that we don't have to duplicated it from pipeline.

**Review by bearfact — CHANGES_REQUESTED** [review]
LGTM lets just commonize any config we can so we aren't maintaining multiple sources.

File: jakelib/utils.js [hunk 2]
```diff
@@ -83,6 +83,26 @@ const replacer = (value, variables) => {
...
+    prod: {
+        lv : { protected: true },
+    },
+    stag: {
+        pp: { protected: true },
+        one: { protected: false },
+        two: { protected: false },
+        qe: { protected: false },
+        uat: { protected: false },
+    },
```
Comment (aprakash-gc) [inline jakelib/utils.js:96]: i got rid of `eph` from this object, i didn't see it being used at all

**Review by bearfact — APPROVED** [review]

**Comment by bearfact** [comment]
:+1:

File: jakelib/utils.js (hunk 2 above)
Comment (aprakash-gc) [reply jakelib/utils.js:96]: @tom-con maybe you know better

Comment (tom-con) [reply jakelib/utils.js:96]: @aprakash-gc won't that stop people from being able to deploy to ephemeral environments?

**Review by tom-con — CHANGES_REQUESTED** [review]

**Review by tom-con — APPROVED** [review]

File: jakelib/scripts.jake [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    'shipping-platform': {
+        service: 'shipping-api'
+    },
+};
+
+namespace('scripts', () => {
+    desc('Script runner | [\'env\',\'cluster\',\'repo\',\'branch\',\'script name\',\'args\'] | `scriptname` is path to script under the `scripts/` dir. `args` are optional script args')
+    task('run', { async: true }, async (environment, cluster, repo, branch, scriptName, scriptArgs) => {
+        if (environment !== 'eph') {
+            let protekted = false
```
[the remark this answers is not in the document]
Comment (aprakash-gc) [reply jakelib/scripts.jake:23]: Intellij complains `protected` is a reserved word. Can JS classes have protected fields/methods?

**Pull request merged**